src/index.html

```diff
@@ -1,14 +1,39 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
-</html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header class="header">
+      <a href="#">
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <a href="#2" class="nav__link is-active">APPLE</a>
+        <a href="#3" class="nav__link is-active">SAMSUNG</a>
+        <a href="#4" class="nav__link is-active">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+        <a href="#5" class="nav__link is-active">НОУТБУКИ И КОМПЬЮТЕРЫ</a>
+        <a href="#6" class="nav__link is-active">ГАДЖЕТЫ</a>
+        <a href="#7" class="nav__link is-active">ПЛАНШЕТЫ</a>
+        <a href="#8" class="nav__link is-active">ФОТО</a>
+        <a href="#9" class="nav__link is-active">ВИДЕО</a>
```

Comment:
Only first link should have class .is-active, not all links. Because only one link can be active))